### DISCUSSION TITLE
Publication Center considers that the note marked as "home" is always updated, and adds a "gardenEntry" tag every time I publish or modify another note.

### DISCUSSION THREAD

**Discussion by MAstilleroF** [oleeskild/obsidian-digital-garden#565] [Q&A] [answered]
Hello:

I don't know if it's the normal way the plugin works, or a one-off problem with my vault configuration. It is the following:

The (only) note I have marked with the property "dg-home" is always updated, even if I don't modify it. And when I go to see what is modified (by clicking on the icon), then I see that the "gardenEntry" tag is added again and again. It is as if the Publication Center does not recognise that the note is marked as "home" and, therefore, every time I publish or modify ANOTHER note it considers that it also has to update that one, the one marked as "home" (even though it was already marked as "home").

It's not a big problem. But I'm worried that everything collapses when I have published or modified, let's say a hundred notes, and the tag "entryGarden" is repeated a hundred times in that note.

Does anyone have the same problem?

Any ideas? 

I'm working with the latest version of Mac and Obsidian, and with Netlify.

Thanks!

M.

![Screenshot 2024-03-09 at 17 32 06](https://github.com/oleeskild/obsidian-digital-garden/assets/46754031/d8019905-6177-4ba2-86b4-ba8e9a754bb7)
![Screenshot 2024-03-09 at 17 30 59](https://github.com/oleeskild/obsidian-digital-garden/assets/46754031/7d88aa17-fd4e-47d3-a28c-388fc6f6839f)

**Comment by MAstilleroF** [comment, marked as answer]
Finally _I think_ I have discovered the source of the problem after a thousand and one trial and error, but not the solution (although there is a workaround).

The note I chose as "home" included a GIF (archived in the vault). I thought that as this file was dynamic, the note had to be updated every time I posted something. So I removed it, and the problem disappeared when the note was just text. Then, I put a JPG image (also archived in the vault), and the problem reappeared.

My **conclusion** is that every time the "home" note is published, if it is the case that it includes an image from the vault, the note has to go to check it there where it is archived, since the note does not have the image, but it is inserted in the image with a link _ad intra_ the vault. And that check is detected by the plugin as a modification. In fact, although the plugin forces you to publish it as a modification, in Netlify nothing happens, of course.

The _pragmatic solution_ is (i) either not to include any image at all, or (ii) to unpublish the home note and publish it periodically to avoid the #gardenEntry tag accumulating hundreds of times.

Well, I don't know, that's what I've come up with and I'll share it with you in case someone else has the same problem.

**Comment by City-of-Memphis-Wastewater** [comment]
Tangential: Looking at the tags like gardenEntry, have you found a way to edit the site title, by editing the github repo directly, rather than via the Digital Garden plugin setting page in Obsidian?  Github isn't great for text search, and I cannot find an instance of my site title string within the repo.

On that note - what will break the plugin? If, say, i edit notes directly in the github browser, rather than in my file system.
